package.json

```diff
@@ -35,7 +35,7 @@
   },
   "dependencies": {
     "@edx/brand": "github:nelc/brand-openedx#open-release/redwood.nelp",
-    "@edx/frontend-component-header": "npm:@edunext/frontend-component-header@5.0.2-alpha.1-nelp.2",
+    "@edx/frontend-component-header": "npm:@edunext/frontend-component-header@5.0.2-alpha.1-nelp.4",
```

Comment:
What happen with 3 xD

Anyway is ok for me

Reply:
That version had an error